velox/docs/configs.rst

```diff
@@ -164,4 +164,16 @@ data size exceeds this limit, otherwise it spills the partition with most data.
 If the limit is zero, then the spiller always spills a previously spilled
 partition if it has any data. This is to avoid spill from a partition with a
 small amount of data which might result in generating too many small spilled
-files.
\ No newline at end of file
+files.
+
+Spark-specific configuration
+----------------------------
+
+``legacySizeOfNull``
```

Comment:
I wonder if it makes sense to add "spark." prefix to all Spark-specific configuration properties for clarity. Also, it would be nice to rename this property to match naming convention used for other properties: "spakr.legacy-size-of-null".

Reply:
Yes, I think it is necessary. fixed it.